packages/app-project/src/screens/ProjectHomePage/components/Hero/components/WorkflowMenu/WorkflowMenu.stories.js

```diff
@@ -1,85 +0,0 @@
-import asyncStates from '@zooniverse/async-states'
-import zooTheme from '@zooniverse/grommet-theme'
-import { Grommet } from 'grommet'
-import { Provider } from 'mobx-react'
-import { mockWorkflow as mockGroupedWorkflow } from '@shared/components/SubjectSetPicker/helpers'
-import WorkflowMenu from './WorkflowMenu'
-
-const store = {
-  project: {
-    background: {
-      src: 'https://panoptes-uploads.zooniverse.org/production/project_background/260e68fd-d3ec-4a94-bb32-43ff91d5579a.jpeg'
-    },
-    description: 'Learn about and help document the wonders of nesting Western Bluebirds.',
-    display_name: 'Nest Quest Go: Western Bluebirds',
-    slug: 'brbcornell/nest-quest-go-western-bluebirds',
-    workflow_description: `Choose your own adventure! There are many ways to engage with this project:  
-      1) "Nest Site": Smartphone-friendly, helps us understand where Western Bluebirds build their nests.  
-      2) "Location": Smartphone-friendly, series of questions on the geographic location of the nest.  
-      3) "Nest Attempt: Smartphone-friendly, for data-entry lovers to record nest attempt data on cards.  
-      4) "Comments": For transcription lovers, we ask you to transcribe all the written comments on the cards.`
-  },
-  user: {
-    loadingState: asyncStates.success,
-    personalization: {
-      projectPreferences: {}
-    }
-  }
-}
-
-const WORKFLOWS = [
-  {
-    completeness: 0.65,
-    default: false,
-    displayName: 'The Family and the Fishing Net',
-    id: '12345'
-  },
-  {
-    completeness: 0,
-    default: false,
-    displayName: 'Games Without Frontiers',
-    id: '7890'
-  },
-  {
-    completeness: 0.99,
-    default: false,
-    displayName: 'Shock The Monkey',
-    id: '5678'
-  }
-]
-
-function StoryContext (props) {
-  const { children, theme } = props
-
-  return (
-    <Grommet
-      background={{
-        dark: 'dark-1',
-        light: 'light-1'
-      }}
-      theme={theme}
-      themeMode={(theme.dark) ? 'dark' : 'light'}
-    >
-      <Provider store={store}>
-        {children}
-      </Provider>
-    </Grommet>
-  )
-}
-
-export default {
-  title: 'Project App / Screens / Project Home / Workflow Menu',
-  component: WorkflowMenu,
-  args: {
-    dark: false,
-    workflows: [ ...WORKFLOWS, mockGroupedWorkflow ]
```

Comment:
`mockedGroupWorkflow` here adds set selection to the story. That should be moved to the Hero component, if this story is being deleted. Otherwise, we don't have a story for subject set selection on the project home page.
https://zooniverse.github.io/front-end-monorepo/@zooniverse/fe-project/index.html?path=/story/project-app-screens-project-home-workflow-menu--default

Reply:
I restored this story, but renamed it to make it clear what it was for. I still didn't catch this when I did my side-by-side comparison previously.

Reply:
It's really confusing that we have two stories for the same thing. You're right about consolidating them in one story.